packages/units/src/range.js

```diff
@@ -42,26 +42,24 @@ export const lerp = (progress, range) =>
  */
 export const progress = (v, range) => {
   const clamped = clamp(v, range);
-  const delta = clamped - range[0];
-  const difference = range[1] - range[0];
+  const delta = clamped - range.MIN;
+  const difference = range.MAX - range.MIN;
 
   return Math.abs(delta / difference);
 };
 
 /**
- * Takes a value and a range in the form of either a tuple and clamps the value to that range.
+ * Takes a value and a range in the form of a tuple and clamps the value to that range.
  *
  * @param {number} value - value to be clamped
- * @param {[number, number]|{MIN: number, MAX: number}} range - range in the form of a tuple.
+ * @param {Object} range - range in the form of a tuple.
+ * @param range.MIN
+ * @param range.MAX
  * @return {number} - number within range
  */
-export const clamp = (value, range) => {
-  const lowerBound = Array.isArray(range)
-    ? Math.min(range[0], range[1])
-    : range.MIN;
-  const upperBound = Array.isArray(range)
-    ? Math.max(range[0], range[1])
-    : range.MAX;
+export const clamp = (value, { MIN, MAX }) => {
+  const lowerBound = Math.min(MIN, MAX);
+  const upperBound = Math.max(MIN, MAX);
```

Comment:
This looks weird with `MIN` and `MAX` object but don't know why some tests are doing inverse value testing here and if removing it would create any side effect https://github.com/GoogleForCreators/web-stories-wp/blob/pacakges/utilities/packages/units/src/test/range.js and they fail without it.

Reply:
Agreed, that is weird. Doing it for the array notation makes sense, but not with `MIN` and `MAX` object.
So I'd say we can remove those tests